src/wizard/javascript/remix/with-error-monitoring.md

```diff
@@ -0,0 +1,94 @@
+---
+name: Remix
+doc_link: https://docs.sentry.io/platforms/javascript/guides/remix/
+support_level: production
+type: framework
+---
+
+## Install
+Sentry captures data by using an SDK within your application’s runtime.
+
+```bash
+# Using yarn
+yarn add @sentry/remix
+
+# Using npm
+npm install --save @sentry/remix
+```
+
+## Configure
+Import and initialize Sentry in your Remix entry points for both the client and server:
+
+```javascript
+import { useLocation, useMatches } from "@remix-run/react";
+import * as Sentry from "@sentry/remix";
+import { useEffect } from "react";
+
+Sentry.init({
+  dsn: "___DSN___",
+});
+```
+
+Initialize Sentry in your entry point for the server to capture exceptions and get performance metrics for your [`action`](https://remix.run/docs/en/v1/api/conventions#action) and [`loader`](https://remix.run/docs/en/v1/api/conventions#loader) functions. You can also initialize Sentry's database integrations, such as Prisma, to get spans for your database calls:
+
+```javascript
+import { prisma } from "~/db.server";
+
+import * as Sentry from "@sentry/remix";
+
+Sentry.init({
+  dsn: "___DSN___",
```

Comment:
Prisma here on errors only?

Reply:
Prisma is only for perf - changed with c5b4fc018afe830d3ac16688b9a0fb8b2a4b1cd5